lib/OpenQA/Schema/Result/Comments.pm

```diff
@@ -90,9 +90,15 @@ our @ISA = qw/Text::Markdown/;
 sub _DoAutoLinks {
     my ($self, $text) = @_;
 
+    # auto-replace every http(s) reference which is not already either html
+    # 'a href...' or markdown link '[link](url)'
+    $text =~ s{(?<!['"(])(http[s]?://[^\s]*)}{<a href="$1">$1</a>}gi;
```

Comment:
I don't get what you mean. What do you expect to change then? IIUC currently `$self->SUPER…` as the last call returns the string which then itself as the last statement is returned by the openQA _DoAutoLinks. So what gives?